internal/service/advancedcluster/resource_advanced_cluster_state_upgrader_test.go

```diff
@@ -26,7 +26,7 @@ func TestMigAdvancedCluster_empty_advancedConfig(t *testing.T) {
 							},
 						},
 						"provider_name": "AWS",
-						"region_name":   "US_EAST_1",
+						"region_name":   "US_WEST_2",
```

Comment:
Any reason for the change of region?

Reply:
as we're running multiple clusters in the same project there is a cross-region limit so i'm using the sam eregion to avoid that